packages/interface-ipfs-core/src/utils/ipfs-options-websockets-filter-all.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const WS = require('libp2p-websockets')
+const filters = require('libp2p-websockets/src/filters')
+const transportKey = WS.prototype[Symbol.toStringTag]
+
+module.exports = () => ({
```

Comment:
this could be just an object since it doesn't take any input, it would reduce LoC in the others files

Reply:
The main reason I created a function here is to enable the options to be customized from the caller without modifying the actual content thanks to the new reference. We had issues in the past in libp2p tests, where its base configuration used in tests was modified in some tests, which caused other tests to have different configurations than expected required.
We currently only use it directly and I can turn that into an object. What do you think?

Reply:
sounds good maybe freeze it to be safe if it makes sense.